airflow/providers/trino/hooks/trino.py

```diff
@@ -81,10 +86,14 @@ def get_conn(self) -> Connection:
                 delegate=_boolify(extra.get('kerberos__delegate', False)),
                 ca_bundle=extra.get('kerberos__ca_bundle'),
             )
+            
+        if _boolify(extra.get('impersonate_as_owner', false)):
```

Comment:
I think you have a typo here, `False`, and not `false`, this is breaking tests.

Reply:
Thanks for pinging this issue, totally forget about that I have raised a PR ;)